### DISCUSSION TITLE
Automatically remove studies not reporting on randomized controlled trials (RCT)

### DISCUSSION THREAD

**Discussion by ntluong95** [asreview/asreview#1604] [Ideas]
Covidence has just released their new feature: Automatically remove studies not reporting on randomized controlled trials (RCT)
https://www.youtube.com/watch?v=dlJp5QS5vrY

Is this possible to implement the same thing on AS Review?

**Comment by Rensvandeschoot** [comment]
You could test if the classifier doesn't already present only papers with RCTs if you only include these in your training data.

**Reply by ntluong95** [reply]
What do you mean? Can you explain it further?

**Reply by Rensvandeschoot** [reply]
ASReview works with [Active Learning](https://asreview.nl/blog/active-learning-explained/), where the classifier tries to predict the most likely relevant records based on the training data. After each labeling decision, the training data becomes bigger, and the model prediction becomes more accurate. If you select a few abstracts where RCTs have been used as prior knowledge, and if you label only RCTs as relevant, the model will most likely learn from your decisions and only present RCTs to label.

**Reply by ntluong95** [reply]
well if you watch the example in Youtube, Covidence automatically make the decision to remove non-RCT for us, with the precision of 99.5%. That's why I am wondering if ASreview can implement the same thing?

**Reply by Rensvandeschoot** [reply]
Ah, so the Covidence tries to predict papers using an RCT and removes these papers as part of the pre-processing phase. Active learning-based tools, like ASReview, move the same papers to the end of the ranking but keep the papers in the dataset. Do you have a labeled dataset where it is known which papers are RCTs? Then, we can run some tests to see if there is any difference in the outcome between both methods.
